src/App.tsx

```diff
@@ -90,56 +96,93 @@ class App extends React.Component<AppProps> {
     );
   };
 
+  detectMob() {
+    const toMatch = [
+      /Android/i,
+      /webOS/i,
```

Comment:
what's webOS? jw

Reply:
Not sure tbh, that's just what one of the things stackoverflow told me to include in my regex for detecting mobile